packages/adyen-integration/src/adyenv3/adyenv3-payment-strategy.ts

```diff
@@ -460,13 +461,22 @@ export default class Adyenv3PaymentStrategy implements PaymentStrategy {
             ? this._cardVerificationComponent
             : this._paymentComponent;
 
+        const isEmptyString = (value: string) => value.toString().trim().length === 0;
+        const isShorterThan3 = (value: string) => value.toString().replace(/ /g, '').length <= 3;
```

Comment:
Correct me if I am wrong, but the variable name does not always mean that the function do. I mean, that 3 is not shorter than 3.

So in this case, its better to rename a variable, or fix the implementation of the function:
```suggestion
        const isShorterThan3 = (value: string) => value.toString().replace(/ /g, '').length < 3;
```